js/bootstrap-datepicker.js

```diff
@@ -2034,3 +2038,13 @@
 	});
 
 }));
+
+function deprecate(msg) {
+	var console = window.console;
+	if (console && console.warn) {
+		console.warn('DEPRECATED: ' + msg);
```

Comment:
Not your fault, be sorry for the poor souls that committed mixed content... and the one that merged it 👿